test/specs/components/action.spec.jsx

```diff
@@ -94,6 +87,9 @@ xdescribe('Action', () => {
 
         sandbox.stub(conversationService, 'postPostback');
         conversationService.postPostback.resolves();
+
+        sandbox.stub(appUtils, 'getIntegration');
```

Comment:
is it necessary to stub this? why can't we just set up the app with the proper integrations?